yucca/training/data_loading/YuccaDataset.py

```diff
@@ -42,7 +42,7 @@ def keep_in_ram(self):
         if len(self.all_cases) < 50:
             self._keep_in_ram = True
         else:
-            print("Large dataset detected. Will not keep cases in RAM during training.")
+            # print("Large dataset detected. Will not keep cases in RAM during training.")  # Do we really need to log this every time?
```

Comment:
This will only be called once right? You could make it a logging.debug so it can be toggled on occasioanlly